task/sbom-json-check/0.1/sbom-json-check.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sbom-json-check
+spec:
+  description: >-
+    Check the syntax of the sbom-cyclonedx.json file which should be found
+    in the /root/buildinfo/content_manifests/ directory
+  params:
+    - name: IMAGE_URL
+      description: the fully qualified image name to be verified
+      type: string
+  results:
+    - description: Test output
+      name: HACBS_TEST_OUTPUT
+  workspaces:
+    - name: workspace
+  steps:
+  - name: sbom-json-check
+    image: quay.io/redhat-appstudio/hacbs-test:latest
+    workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)
+    securityContext:
+      runAsUser: 0
+      capabilities:
+        add:
+          - SETFCAP
+    volumeMounts:
+      - mountPath: /shared
+        name: shared
+    script: |
+      cd /shared
+      touch fail_result.txt
+      if [ -f "sbom-cyclonedx.json" ]
+      then
+        result=$(echo -n $(cyclonedx-linux-x64 validate --fail-on-errors --input-file sbom-cyclonedx.json))
```

Comment:
Do we need the `--fail-on-errors` option set if we're checking the output of the validation in the `$result`?

Reply:
Yes, we can remove it

Reply:
Updated.

Reply:
Thanks!